src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Given_xBind_Binding.cs

```diff
@@ -43,5 +43,103 @@ public void When_StateTrigger()
 
 			Assert.AreEqual("Updated!", SUT._StringField.Text);
 		}
+
+		[TestMethod]
+		public void When_Simple_TwoWay()
+		{
+			var SUT = new Binding_Simple_TwoWay();
+
+			Assert.AreEqual(0, SUT.MyIntProperty);
+
+			SUT.ForceLoaded();
+
+			Assert.AreEqual(0, SUT.myObject.MyProperty);
+			Assert.AreEqual(0, SUT.MyIntProperty);
+
+			SUT.MyIntProperty = 1;
+
+			Assert.AreEqual(1, SUT.myObject.MyProperty);
+
+			Console.WriteLine("SUT.myObject.MyProperty = 2");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5087470)